index.css

```diff
@@ -0,0 +1,361 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif !important;
```

Comment:
why are you using `!important`, this font family is it changed some where in the code . instead I can suggest to create defferent modifier for different font family.

if this font family is not changed so you don't need `!important`.